assets/templates/dining_lists/dining_slot_allergy.html

```diff
@@ -4,34 +4,29 @@
 
 
 {% block details%}
-    <div class="row col-12 m-0 p-0 p-md-2 py-2 text-size-3">
-        <div class="{% if can_delete_all or can_delete_some %} col-10 col-sm-11 {% else %} col-12 {% endif %}">
+    <div class="row col-12 m-0 p-0 p-md-2 py-2">
+        <div class="col-12">
             <div class="row pr-2">
-                <div class="col-12 col-md-6 p-0 show-md">Name</div>
+                <div class="col-12 col-md-6 p-0 show-md"><b>Name</b></div>
                 <div class="col-12 col-md-6 pr-3">
-                    Allergy
+                    <b>Allergy</b>
                 </div>
             </div>
         </div>
-        {% if can_delete_all or can_delete_some %}
-            <div class="col-2 col-sm-1 p-0 text-center">Remove</div>
-        {% endif %}
     </div>
-    <div class="col-12 p-0">
+    <div class="row col-12 m-0 p-0 p-md-2 py-2">
```

Comment:
Ik heb 't niet geprobeerd maar ik zou denken dat `col-12 m-0 p-0` niet nodig is omdat `col-12` ook al op de child elements zit.

Reply:
Row, m-0 en p-0 zijn nu verwijderd, heeft inderdaad hetzelfde effect. m-0 en p-0 waren om row uit te cancellen, maar weet niet waarom row erbij stond.